README.md

```diff
@@ -19,3 +25,7 @@ PhysFS::Util namespace.
  - Both ifstream and ofstream are standard streams, and only have a couple of
 extra methods - `eof` and `length`, which call `PHYSFS_eof` and 
 `PHYSFS_fileLength` respectively.
+
```

Comment:
I like all other changes in this file, except for build section, because building really depends on how you're trying to integrate the lib. If you rephrased to give this as an example, and maybe point to general CMake build directions, that'd be much better.